pom.xml

```diff
@@ -235,11 +215,6 @@
             </licenseMerges>
           </configuration>
         </plugin>
-        <plugin>
-          <groupId>org.sonatype.plugins</groupId>
-          <artifactId>jarjar-maven-plugin</artifactId>
```

Comment:
This plugin is discontinued, I don't think the parent pom should encourage its use. If some projects are using it, they should probably move to the Maven Shade plugin.

Reply:
This would break only sonar-pli if also applied to parent